support-frontend/typescript-errors.json

```diff
@@ -4,19 +4,19 @@
             "TS1208": 1,
             "TS2345": 14,
             "TS7006": 4,
+            "TS2322": 8,
```

Comment:
The changes here seem to be down to not having the latest version of the TS errors file on main

Reply:
maybe run `yarn` to install latest deps? I think the errors should go down from the main branch

Reply:
Yep that did it!